docs/theory/diffusion/intro.rst

```diff
@@ -0,0 +1,140 @@
+the units of the diffusion coefficients are :math:`length^{2}/s`. For 1D diffusion along DNA, D is often expressed in :math:`µm^{2}/s` or :math:`kbp^{2}/s` :cite:`Kochaniak2009`.
+
+Methods for obtaining the diffusion coefficient
+------------------------------------------------
+
+The first step for analyzing a diffusive protein from a kymograph, is to track each binding event. The resuling tracks give the positions of each bound protein over time.
+These tracks can then be analyzed further to determine the diffusion coefficient. Below we discuss two methods for diffusion analysis that are available in Pylake. 
+
+Mean Squared Displacement analysis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A common method for determining the diffusion coefficient, is to first compute the mean squared displacement (MSD) as a function of lag time for each track. 
+For purely diffusive motion, the first part of the MSD curve is linear, with the slope proportional to the diffusion coefficient. For 1D diffusion the MSD is given by: :math:`MSD = 2D\Delta t`.
+The diffusion coefficient can be obtained by fitting a linear function to the MSD curve, as illustrated in the figure below (simulated data):
+
+.. image:: figures/MSD_analysis.png
+
+Note that for the MSD curve, the 'lag time' (aka 'time interval') is given on the x-axis, rather than 'time'. 
+To compute the MSD for a certain lag time, you scan over a track, one data point at a time and compute the displacement squared 
+from that time until that time + the lag time. So each point on the MSD curve, is an average of displacements for a specific lag time, see the :doc:`theory section on diffusive motion</theory/diffusion/diffusion>`
+for the equation for the MSD. 
+
+The largest possible lag time is the duration of a track and the shortest possible lag is the time resolution, the line time of the kymograph. 
+The image above does not show the complete MSD curve. The longest possible lag time would be 2500 seconds, but the MSD curve is only given up to a lag time of 5 seconds. 
+The full MSD curve looks as follows:
+
+.. image:: figures/MSD_full.png
+
+As you can see, the full MSD curve becomes less and less linear for larger lags times. The funky shape of the MSD curve for large lag times is caused by:
+
+#. The MSD at shorter time lags is averaged over more lags and therefore more accurate
+
+#. Consecutive data points in the MSD are highly correlated
+
+The MSD curve for larger lags is not suitable for a linear fit. An important step in MSD analysis, is therefore to determine which number of lags is optimal for a linear fit and obtaining the diffusion coefficient.
+Taking too many lags, may result in an MSD curve that is not fully linear, while taking too little lags may not give enough data points for a good estimate of the diffusion coefficient.
```

Comment:
```suggestion
Taking too many lags may result in an MSD curve that is not fully linear, while taking too few lags may not give enough data points for a good estimate of the diffusion coefficient.
```

Reply:
Interestingly, if there is no observation noise and free diffusion, `2` lags is optimal.